services-add-api-oauth.md

```diff
@@ -0,0 +1,70 @@
+# Add API to project using OAuth
+
+Adding an API to an empty project is the same whether you are working in a personal or enterprise project. Adding an API to a templated project is similar, with one small variation: APIs are added to individual workspaces, not to the project as a whole.
+
+To begin adding an API from within a templated project, first select the appropriate workspace to open the *Workspace overview*. Then, select **+ Add Service** in the left navigation and choose **API** from the dropdown. 
+
+In an empty project, select **+Add to Project** in the left navigation of the *Project overview* and then choose **API**, or select **Add API** from the quick start buttons.
+
+![](images/services-add-to-project.png)
+
+## Add an API
+
+Using REST APIs allows your application to make calls to Adobe services and products. The *Add an API* dialog shows a list of available services with the default *View by* setting to show only those services available to you.
+
+> **Note:** Many services are only available through paid licenses or subscriptions. Licenses and subscriptions can refer to either your organization or your personal licenses if you are building a personal project. 
+>
+> For this reason, if you select "All" from the *View by* dropdown, you may notice that several services appear greyed out in the list. If you believe that you should have access to one of these disabled services, please speak with your system administrator or Adobe sales representative.
+
+![](images/services-add-api.png)
+
+Once you have found and chosen an API that you would like to add, select **Next** to begin configuring the API.
+
+![](images/services-api-oauth-select.png)
+
+## Configure API
+
+User authentication using OAuth allows your end users to sign in to your integration with an Adobe ID. With an OAuth token, your integration will be able to access Adobe services and content on behalf of the logged in user.
+
+To configure an API using OAuth 2.0 authentication and authorization, you must first select the platform where you want to use this integration: Web, iOS, or Android. 
```

Comment:
can we please change to: "...select the platform**s** where you..."
the developer is able to select multiple platforms at once